db/migrate/20160331232622_devise_create_users.rb

```diff
@@ -1,42 +0,0 @@
-class DeviseCreateUsers < ActiveRecord::Migration
-  def change
-    create_table(:users) do |t|
```

Comment:
Every time I pulled in the remote master it showed that...I'll pull the new master and delete everything else so it stops happening!  Or do you have another way to make it stop?